marklogic-data-hub/build.gradle

```diff
@@ -252,11 +252,13 @@ def propFile = file("gradle.properties")
 String fileContents = propFile.text
 boolean sslRun = Boolean.parseBoolean(System.properties['ssl'])
 boolean certAuth = Boolean.parseBoolean(System.properties['certauth'])
+String env = environmentName
 def props = new Properties()
 propFile.withInputStream { props.load(it) }
 
 task bootstrap(type: JavaExec) {
 	dependsOn 'setupSSL'
+    systemProperty "environmentName", env
```

Comment:
this should not be required, seems very hacky.  you could pass an argument to the main method instead of using a system property.

Reply:
Shall I use args = ["$environmentName"] instead in the bootstrap task?